example/provider-local/shoot-ipv6.yaml

```
@@ -7,21 +7,15 @@ metadata:
     shoot.gardener.cloud/infrastructure-cleanup-wait-period-seconds: "0"
     shoot.gardener.cloud/cloud-config-execution-max-delay-seconds: "0"
 spec:
-  seedName: local
   cloudProfileName: local
   secretBindingName: local # dummy, doesn't contain any credentials
   region: local
   networking:
     type: calico
-    pods: 2002:db8:1::/48
-#    nodes: 2002:db8:2::/64 -> it doesn't make sense to specify this, node IPs are given by the seed cluster's pod CIDR
-    services: 2002:db8:3::/108
     ipFamilies:
     - IPv6
-    # TODO(scheererj): Drop this once v1.32 has been released and https://github.com/gardener/gardener-extension-networking-calico/pull/250 is available as release
-    providerConfig:
-      apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
-      kind: NetworkConfig
+    pods: 2002:db8:1::/48
+    services: 2002:db8:3::/108
```

Please use the documentation range instead of the `6to4` range:

```suggestion
    pods: 2001:db8:1::/48
    services: 2001:db8:3::/108
```